google-cloud-spanner/src/test/java/com/google/cloud/spanner/SessionImplTest.java

```diff
@@ -515,4 +560,11 @@ public void multiUseReadOnlyTransactionReturnsMissingTransactionId() throws Pars
             () -> txn.readRow("Dummy", Key.of(), Collections.singletonList("C")));
     assertEquals(ErrorCode.INTERNAL, e.getErrorCode());
   }
+
+  private com.google.protobuf.Timestamp getCurrentGoogleTimestamp() {
```

Comment:
I think that `com.google.cloud.Timestamp.now().toProto() ` produces the same

Reply:
Changed.